security/src/main/java/nl/nn/adapterframework/lifecycle/servlets/ServletAuthenticatorBase.java

```diff
@@ -15,36 +15,58 @@
 */
 package nl.nn.adapterframework.lifecycle.servlets;
 
+import java.util.Arrays;
+import java.util.Collections;
 import java.util.HashSet;
 import java.util.List;
+import java.util.Properties;
 import java.util.Set;
 
+import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.springframework.beans.factory.config.ConfigurableListableBeanFactory;
 import org.springframework.context.ApplicationContext;
 import org.springframework.context.ApplicationContextAware;
 import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.core.env.MapPropertySource;
+import org.springframework.core.env.PropertySource;
+import org.springframework.core.env.PropertySources;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.web.SecurityFilterChain;
 
 import lombok.Getter;
-import nl.nn.adapterframework.lifecycle.ServletManager;
-import nl.nn.adapterframework.util.LogUtil;
 
 public abstract class ServletAuthenticatorBase implements IAuthenticator, ApplicationContextAware {
 	private static final String HTTP_SECURITY_BEAN_NAME = "org.springframework.security.config.annotation.web.configuration.HttpSecurityConfiguration.httpSecurity";
+	public static final List<String> DEFAULT_IBIS_ROLES = Collections.unmodifiableList(Arrays.asList("IbisObserver", "IbisAdmin", "IbisDataAdmin", "IbisTester", "IbisWebService"));
```

Comment:
Perhaps think about avoiding the 'IBIS' name with new code? `DEFAULT_FF_ROLES`?

Reply:
Ha! You're absolutely right, but I think the word Ibis might have a few more occurrences that we would like.
The word Ibis is not affiliated with WeAreFrank so there's no FOSS issue there, but it would be good to fully migrate to our new name.